framework/src/utils/InputParameters.C

```diff
@@ -160,6 +160,16 @@ InputParameters::operator+=(const InputParameters & rhs)
 
 void
 InputParameters::addCoupledVar(const std::string & name, Real value, const std::string & doc_string)
+{
+  // std::vector<VariableName>(1, Moose::stringify(value)),
+  addParam<std::vector<VariableName>>(name, doc_string);
+  _coupled_vars.insert(name);
+  _params[name]._coupled_default.assign(1, value);
+  _params[name]._have_coupled_default = true;
```

Comment:
why not make _have_coupled_default a vector of bool?

Reply:
I don't need it and it makes more changes in `Coupleable` necessary. The way `CoupleAble` checks for default vs. variable names provided operates on the whole vector provided by the user.